org.eclipse.m2e.profiles.core/OSGI-INF/org.eclipse.m2e.profiles.core.internal.management.ProfileManager.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When this change is submitted after you released Tycho 2.7.2 (which I assume since you used the `classpath=true` setting), then this file can be ignored

Reply:
Whichever happens first^^

Reply:
This file can now be ignored.